modular_zubbers/code/modules/loadout/categories/accessories.dm

```diff
@@ -1 +1,8 @@
-// THIS IS WHERE LOADOUT DATUMS GO FOR ACCESSORIES (REMOVE THIS COMMENT AND TICK THIS FILE IF YOU ADD ANYTHING)
+/datum/loadout_item/accessory/pocketwatch
```

Comment:
did you tick the file? only one file changed

Reply:
I think so? I honestly haven't done anything like this before.

Reply:
This is my first ever pull request that I've ever made XD

Reply:
are you editing this in vscode? if so, hit the unticked button at the bottom right of the screen when on this file

if on DM, i dunno

Reply:
Am on DM, and its ticked on there....

Reply:
then its already ticked and this isnt an issue

Reply:
i think so